rcsb/utils/chemref/RcsbLigandReferenceGenerator.py

```diff
@@ -0,0 +1,300 @@
+##
+#  File:           RcsbLigandReferenceGenerator.py
+#  Date:           2025-12-10 Chenghua Shao
+#
+#  Update:
+##
+"""
+Generate ligand quality reference data that is used for ligand quality score computation.
+
+"""
+import csv
+import logging
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+from rcsbapi.data import DataQuery as Query
+from rcsbapi.data import ALL_STRUCTURES
```

Comment:
Can you commit the following suggestion? Something weird is happening with the linting, which I'll need to look into later.

```suggestion
from rcsbapi.data import ALL_STRUCTURES  # pylint: disable=no-name-in-module
```